java/pgv-java-stub/src/main/java/io/envoyproxy/pgv/StringValidation.java

```diff
@@ -19,6 +19,8 @@
     private static final int UUID_DASH_3 = 18;
     private static final int UUID_DASH_4 = 23;
     private static final int UUID_LEN = 36;
+    private static final Pattern HTTP_HEADER_NAME_PATTERN = Pattern.compile("^:?[0-9a-zA-Z!#$%&'*+-.^_|~\u002C]+$");
+    private static final Pattern HTTP_HEADER_VALUE_PATTERN = Pattern.compile("^[ \t]*(?:[\u0020-\u007E\u0080-\u00FF](?:[ \t]+[\u0020-\u007E\u0080-\u00FF])?)*[ \t]*$");
```

Comment:
Do we need to specify the regex for every language? It would be ideal if there was a single source-of-truth to avoid potential inconsistency.